sklearn/metrics/_ranking.py

```diff
@@ -352,7 +364,7 @@ def roc_auc_score(y_true, y_score, *, average="macro", sample_weight=None,
             Machine Learning, 45(2), 171-186.
             <http://link.springer.com/article/10.1023/A:1010920819831>`_
 
-    See also
+    See Also
```

Comment:
Are you sure about this?

```
$ git grep "See also$" | wc -l
173
$ git grep "See Also$" | wc -l
70
```

Reply:
This is inside the numpydoc validation :)